lib/common/common/ByteSwap.h

```diff
@@ -50,6 +48,22 @@ RtlUlonglongByteSwap(
     _In_ ULONGLONG Source
     );
 #endif
-#endif
 
+#elif defined(__APPLE__)
+#include <libkern/OSByteOrder.h> 
+
+#define RtlUshortByteSwap(_x)    OSSwapInt16((_x))
+#define RtlUlongByteSwap(_x)     OSSwapInt32((_x))
+#define RtlUlonglongByteSwap(_x) OSSwapInt64((_x))
+
+#elif defined(__linux__)
```

Comment:
Similar question about ```__linux__```. https://github.com/dotnet/coreclr/blob/master/CMakeLists.txt does defining ```__LINUX__```. Maybe we need to add that to our CMakeLists.txt. Would ```__LINUX__``` also work for you?